src/Wallabag/UserBundle/Resources/translations/wallabag_user.de.yml

```diff
@@ -0,0 +1,11 @@
+# Two factor mail
+auth_code:
+    on: 'an'
+    mailer:
+        subject: 'Wallabag Authentifizierungcode'
```

Comment:
not on the 2.2 branch https://github.com/wallabag/wallabag/blob/2.2/src/Wallabag/UserBundle/Resources/translations/wallabag_user.en.yml ;-)